editor/plugins/script_editor_plugin.cpp

```diff
@@ -3766,6 +3766,14 @@ void ScriptEditor::unregister_syntax_highlighter(const Ref<EditorSyntaxHighlight
 	syntax_highlighters.erase(p_syntax_highlighter);
 }
 
+void ScriptEditor::update_connected_methods() {
+	ScriptTextEditor *ste = Object::cast_to<ScriptTextEditor>(_get_current_editor());
+
+	if (ste) {
+		ste->_update_connected_methods();
+	}
```

Comment:
```suggestion
	for (int i = 0; i < tab_container->get_tab_count(); i++) {
		ScriptTextEditor *ste = Object::cast_to<ScriptTextEditor>(tab_container->get_tab_control(i));
		if (ste) {
			ste->_update_connected_methods();
		}
	}
```
This should update all scripts, since the connected method may not be in the current editor.
Unless we notice it causes lag or something, it should be fine to update all of them.